### PR TITLE
[Workers] Rephrase the DO requirement

### LINKED ISSUE #7133: Enabling durable objects is unclear
### Which Cloudflare product does this pertain to?

Workers

### Existing documentation URL(s)

https://developers.cloudflare.com/workers/learning/using-durable-objects/#using-durable-objects-1

### Section that requires update

Using Durable Objects

### What needs to change?

The sentence

> To start, enable Durable Objects for your account by logging in to [the Cloudflare dashboard](https://dash.cloudflare.com/) Workers > Durable Objects.

Seems to be unclear. There isn't a section under `Workers` for `Durable Objects`, and instead I found it inside a worker.

### How should it change?

It would be good to know what I'm exactly looking for, or where it might not be. For example, I'm not sure if it's visible if you aren't in a paid worker plan, though I did the upgrade in hopes it would show it for me.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by cicku**
Fix #7133

### CODE REVIEW

**Review by sh471 — COMMENTED** [review]
[Workers] Rephrase the DO requirement #7263

**Comment by sh471** [comment]
sorry

File: content/workers/learning/using-durable-objects.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ For details on the specific Durable Object APIs, refer to the [Runtime API docum
 
 Durable Objects are named instances of a class you define. Like a class in object-oriented programming, the class defines the methods and data a Durable Object can access.
 
-To start, enable Durable Objects for your account by logging in to [the Cloudflare dashboard](https://dash.cloudflare.com/) > **Workers** > **Durable Objects**.
+To start, make sure the [Workers Paid plan](/workers/platform/pricing/#durable-objects) is purchased.
```
Comment (deadlypants1973) [inline content/workers/learning/using-durable-objects.md]:
```suggestion
To start, enable Durable Objects for your account by purchasing Workers Paid:

 1. Log in to [the Cloudflare dashboard](https://dash.cloudflare.com/).
 2. Go to **Workers** and select your Worker. 
 3. In your Worker, scroll down to **Durable Objects** **Learn more** > **View Paid Plan**.
 4. Select **Purchase Workers Paid** and complete the payment process to enable Durable Objects.
```

**Review by deadlypants1973 — APPROVED** [review]

**Comment by deadlypants1973** [comment]
Thank you for PR'ing this, it was on my list but using your PR as an opportunity to rephrase with full instructions :)

**Pull request merged**